akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -392,6 +395,7 @@ private[stream] final class GraphInterpreterShell(
       case Resume(_) ⇒
         if (GraphInterpreter.Debug) println(s"${interpreter.Name}  resume")
         resumeScheduled = false
+        currentLimit = effectiveLimit()
```

Comment:
assuming that it will be used for something, why is it not set in all other cases (apart from AsyncInput) as well?